service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NotifyGatekeeperController.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import io.swagger.annotations.Api;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.fpl.events.NotifyGatekeeperEvent;
+
+@Api
+@RestController
+@RequestMapping("/callback/notify-gatekeeper")
+public class NotifyGatekeeperController {
+
+    private final ApplicationEventPublisher applicationEventPublisher;
+
+    @Autowired
+    public NotifyGatekeeperController(ApplicationEventPublisher applicationEventPublisher) {
+        this.applicationEventPublisher = applicationEventPublisher;
+    }
+
+    @PostMapping("/submitted")
+    public ResponseEntity handleAboutToSubmitEvent(
```

Comment:
Please change to void. It will return 200 status code until exception is thrown. We will apply same thing to rest of the controllers when this story is done.